server/socketCommands.ts

```diff
@@ -52,6 +52,19 @@ const socketCommands = (io: Server)=>{
     return false
   }
 
+  const roomCanStartGame = (roomName: string) => {
+    const room = getRoom(roomName)
+      if(room){
+        const members = room.members
+      
+        for(let member of members) {
+          if(member.readyStatus === false) return false
+        }
+        
+        return true
+      }
```

Comment:
If by writing if(room) you're assuming that it's possible for room to be undefined, then shouldn't you return false outside of the if block for the times when there is no room? And if room will never be undefined, then the if block doesn't matter.